name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/Images.java

```diff
@@ -103,6 +103,7 @@
     POEDITOR_LOGO("poeditor-logo.png"), //$NON-NLS-1$
     DIVVYDIARY_LOGO("divvydiary.com-logo.png"), //$NON-NLS-1$
     EODHISTORICALDATA_LOGO("eodhistoricaldata-logo.png"), //$NON-NLS-1$
+    MYDIVIDENDS24_LOGO("mydividends-logo.png"), //$NON-NLS-1$
```

Comment:
rename logo to "mydividends24.de-logo.png"
Is the logo approved for use and subject to e.g. [Creative Commons CC0](https://creativecommons.org/publicdomain/zero/1.0/legalcode)
Please have a look at the [Contributings](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md) for Portfolio Performance.